tests/elements-regression/src/controls/media.js

```diff
@@ -0,0 +1,70 @@
+const { ControlBase } = require( './control-base' );
+
+class Media extends ControlBase {
+	getSelector() {
+		return `.elementor-control-${ this.config.name }.elementor-control-type-media`;
+	}
+
+	async getValue() {
+		await this.openMediaLibrary();
+
+		const selectedMediaId = await this.getSelectedMediaId();
+
+		const value = selectedMediaId
+			? Object.entries( process.env )
+				.find( ( [ key, id ] ) =>
+					key.startsWith( 'ELEMENTS_REGRESSION_MEDIA_IDS_' ) &&
+					id === selectedMediaId,
+				)
+				?.[ 0 ]
+				?.replace( 'ELEMENTS_REGRESSION_MEDIA_IDS_', '' )
```

Comment:
maybe extract to function to make it in the same high-level abstraction level as other parts of this function?

and maybe instead of creating an env var for each media ID, it'll be easier to store some object? WDYT? I'm not sure about it

Reply:
Done